css/properties/list-style-image.json

```diff
@@ -15,7 +15,8 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "Prior to Firefox 86 this property did not accept an <image> type, and required the url of an image."
             },
             "firefox_android": {
               "version_added": "4"
```

Comment:
Is it likely that the note also applies to Android? If so we may as well copy it over